galasa-managers-parent/galasa-managers-cloud-parent/dev.galasa.kubernetes.manager.ivt/.factorypath

```diff
@@ -0,0 +1,58 @@
+<factorypath>
```

Comment:
Need to remove these files from the repo, add to .gitignore at the base of the repo and delete